apis/Google.Cloud.Datastore.V1/Google.Cloud.Datastore.V1/DatastoreDbImpl.cs

```diff
@@ -142,7 +214,7 @@ public DatastoreDbImpl(string projectId, string namespaceId, DatastoreClient cli
             DatastoreTransaction.Create(Client, ProjectId, NamespaceId, Client.BeginTransaction(ProjectId, callSettings).Transaction);
 
         /// <inheritdoc/>
-        public async override Task<DatastoreTransaction> BeginTransactionAsync(CallSettings callSettings = null)
+        public override async Task<DatastoreTransaction> BeginTransactionAsync(CallSettings callSettings = null)
```

Comment:
Let's split this change and below in a different commit as styling fixes?

Reply:
Its a very small fix and as we are already making changes in this file. I think this should be ok.
let me know what you think?

Reply:
I get your point. However,  the process is to raise a separate commit for styling fixes (if not PR) and then the feature specific changes. This change is not feature specific. Right?

Reply:
Make sense will update it.
Thanks for clarification.